README.md

```diff
@@ -6,9 +6,9 @@ Simplify
 Generic Android Deobfuscator
 ----------------------------
 
-Simplify uses a virtual machine to understand what an app does. Then, it applies optimizations to create code that behaves identically, but is easier for a human to understand. Specifically, it takes Smali files as input and outputs a Dex file with (hopefully) identical semantics but less complicated structure.
+**Simplify** is a framework that utilizes a virtual machine to understand what an app does and then applies optimizations to create code that is easier for a human to understand while maintaining the identical behavior of the application. It takes Smali files as the input and outputs a Dex file with (hopefully) identical semantics but a less complicated/obfuscated structure.
```

Comment:
A framework is something that's incomplete. It's something you fill out with your own stuff. It helps you get started. Simplify isn't a framework. It's a complete tool. The type of tool is a few lines above, so I don't even say "Simplify is a tool ..."

The first paragraph is written in intentionally short, simple, easy to digest sentences with simple words. This is for clarity, and to not scare people away. It's an introduction. I'm continually disappointed and turned off by densely jargonized READMEs. Save that for later. I just want to know what this thing is and how it works, so I treat the readers the same way. Notice "behaves identically", and then later on, near the end, the word "semantics" is used. So the reader should be able to infer what that word means from context. The wording is intentional.

I always avoid using "this/that" because it looks off-the-cuff and lacks clarity. How does it read aloud? "complicated or obfuscated structure"? "complicated and or structure?" To me, "complicated structure" conveys the same meaning, without wasting your time with extra words and ambiguous symbols.